### DISCUSSION TITLE
Installing motion on Birdnet-Pi

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#646] [Q&A]
I am using a "Playstation Eye" https://fr.wikipedia.org/wiki/PlayStation_Eye as my microphone.

-It has 4 microphones, so does it mean I should select 4 in Audio channels (settings on Birdnet-Pi)?

-My second question. I would like to use the camera of this "Playstation Eye" to take pictures at regular intervals. To do this I will need to install motion https://gist.github.com/sidward35/132f2e0e85e505e92712a552cc716324 

Do you think it can cause problems or interfere with Birdnet-Pi?

Thank you.

**Comment by lloydbayley** [comment]
2nd question opinion: I would think the only problem would be processor power and memory absorption. It also depends a lot on how you are going to code the taking of pictures as to the efficiency and resource-intensive'ness. I can't see how motion itself would cause any problems apart from draining resources. If you're worried, image the SD card as a backup and try it.
